import org.apache.hadoop.fs.PathIOException;
public class FunctionUse3699 {
public void funcUse() {
PathIOException pathioexception = new PathIOException();
}
}